sources/AutoFilerTab.cpp

```diff
@@ -39,8 +46,8 @@ AutoFilerTab::AutoFilerTab()
 	BMessage panelMsg(MSG_FOLDER_CHOSEN);
 	fFilePanel = new BFilePanel(B_OPEN_PANEL, NULL, NULL,
 		B_DIRECTORY_NODE, true, &panelMsg, fRefFilter);
-	fFilePanel->Window()->SetTitle("AutoFiler: Add folders");
-	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, "Add");
+	fFilePanel->Window()->SetTitle(B_TRANSLATE("AutoFiler: Add folders"));
+	fFilePanel->SetButtonLabel(B_DEFAULT_BUTTON, B_TRANSLATE("Add"));
```

Comment:
Ok. Will do.

I didn't bring this up, but the translation of one word like "Add" may be different depending on the language context. The translator really needs to see the GUI with his or her translations to make sure it makes sense.